### PR TITLE
Added multisig in Rococo

### LINKED ISSUE #3836: Can we have Multisig feature enabled on Rococo?
I want to reserve ParaId through `Multisig`. `Multisig` is enabled on Kusama. But I want to test it on its testnet Rococo before doing anything on Kusama. 

cc @danforbes @ltfschoen

### PR DESCRIPTION
**Pull request by ayushmishra2005**
resolves #3836

### CODE REVIEW

File: runtime/rococo/src/lib.rs [hunk 1]
```diff
@@ -253,6 +253,7 @@ construct_runtime! {
 
 		// Pallet for sending XCM.
 		XcmPallet: pallet_xcm::{Pallet, Call, Storage, Event<T>, Origin} = 99,
+		Multisig: pallet_multisig::{Pallet, Call, Storage, Event<T>},
```
Comment (bkchr) [inline runtime/rococo/src/lib.rs:256]: Should be moved up under `Proxy`.

**Review by bkchr — APPROVED** [review]

**Comment by bkchr** [comment]
Also requires  a master merge

**Comment by ayushmishra2005** [comment]
> Also requires a master merge

Done @bkchr

**Comment by bkchr** [comment]
bot merge

**Comment by bkchr** [comment]
bot merge

**Pull request merged**